docroot/themes/humsci/humsci_basic/src/scss/_tools.mixins.themes.scss

```diff
@@ -0,0 +1,50 @@
+// Mixin to apply styles to Airy theme
+@mixin airy() {
+  @if $current-theme == 'airy' {
+    @content;
+  }
+}
+
+// Mixin to apply styles to Colorful theme
+@mixin colorful() {
+  @if $current-theme == 'colorful' {
+    @content;
+  }
+}
+
+// Mixin to apply styles to Traditional theme
+@mixin traditional() {
+  @if $current-theme == 'traditional' {
+    @content;
+  }
+}
+
+// Mixin to apply styles to multiple themes
+// Params: (array)
+// E.g.
+// @themes(('airy', 'colorful'))
+// This has a default to fallback to applying to all themes if nothing is passed in
+@mixin themes($theme-list: $global-theme-list) {
```

Comment:
I think there is a case to be made to drop the default value for this parameter, so that if no arguments are included with the mixin, it will results in an error. I'm thinking that if I'm using this mixin and don't include any arguments, it's probably not intentional since I would have just added those styles without a mixin at all. If we remove the default value, an error will be thrown in that case letting us know something is amiss.

Reply:
ok! that sounds fair :) I am realizing too, that since sass is a dev side library, we need to know if there are errors before css compilation anyways so might as well make it error and not fail silently (I was thinking more in line with JS resiliency before)

Reply:
I've changed this and the relevants tests